config/rbac/role.yaml

```diff
@@ -1,119 +1,172 @@
+
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind: Role
 metadata:
   creationTimestamp: null
-  name: manager-role
+  name: jenkins-operator
 rules:
-- apiGroups:
```

Comment:
From down there this file can be skipped as it was also autogenerated - reason for changes.